Source/Extensions/StringExtensions.swift

```diff
@@ -2,59 +2,28 @@ import Foundation
 import UIKit
 
 extension String {
-
-    func getBundle(forClass: AnyClass) -> Foundation.Bundle {
-#if SWIFT_PACKAGE
-        let baseBundle = Bundle.module
-#else
-        let baseBundle = Foundation.Bundle(for: forClass)
-#endif
-        let path = baseBundle.path(forResource: "Frames", ofType: "bundle")
-        // swiftlint:disable:next force_unwrapping
-        return path == nil ? baseBundle : Foundation.Bundle(path: path!)!
+    func localized(comment: String = "") -> String {
+        NSLocalizedString(self, bundle: Bundle.base, comment: "")
     }
 
-    func localized(forClass: AnyClass, comment: String = "") -> String {
```

Comment:
👍 👍 👍 

I remember raising this somewhere but can't find any task or Issue, so might come across it later 🙈.